velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -39,6 +39,34 @@ Expected<std::shared_ptr<DateTimeFormatter>> getDateTimeFormatter(
           std::string_view(format.data(), format.size()));
   }
 }
+
+// Initialize `formatter_` based on the provided format string. It determines
+// formatter type based on the `legacyFormatter` flag. The legacy formatter
+// returns false for an invalid format, while the Joda formatter throws a
+// Velox user error.
```

Comment:
Above description does not match the implementation. Comment suggestion:

> Creates datetime formatter based on the provided format string. When legacy formatter is used, returns nullptr for invalid format; otherwise, throws a user error.

Reply:
Thanks for your valuable input! Fixed.